pkg/client/client.go

```diff
@@ -156,23 +158,33 @@ func parseRevertReason(bz []byte) (string, error) {
 	return string(bz[68 : 68+size.Int64()]), nil
 }
 
+type callLog struct {
+	Address common.Address `json:"address"`
+	Topics  []common.Hash  `json:"topics"`
+	Data    hexutil.Bytes  `json:"data"`
+}
+
+// see: https://github.com/ethereum/go-ethereum/blob/v1.12.0/eth/tracers/native/call.go#L44-L59
 type CallFrame struct {
```

Comment:
Please unexport `CallFrame` (rename it to `callFrame`)

Reply:
Fixed
https://github.com/datachainlab/ethereum-ibc-relay-chain/pull/18/commits/72bb6a14ce78cadc7822037a0c4d51303d14540b